### PR TITLE
Update README.md for running examples on the web

### PR DESCRIPTION
**Pull request by FuriouZz**
Update README.md with more explanation on how to run examples on the web

### CODE REVIEW

**Comment by FuriouZz** [comment]
Good point! I suppose this document will be updated more frequently.

**Comment by rukai** [comment]
Oh awesome!
I tried to run the examples last night, and after following the steps I thought "now what...?"

**Comment by rukai** [comment]
Should `python3 -m http.server` as a webserver be fine?
I cant get it to work
image

File: README.md [hunk 1]
```diff
@@ -46,6 +48,27 @@ RUSTFLAGS=--cfg=web_sys_unstable_apis cargo build --target wasm32-unknown-unknow
...
 ```
 
+Create an `index.html` file into `target/generated` directory and add the following code:
+
+```html
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="google" content="notranslate">
```
Comment (grovesNL) [inline README.md:58]: nit: we could remove this `notranslate` tag and the `title` to make this a bit shorter

**Comment by grovesNL** [comment]
@rukai this won't show all the examples in `index.html`, it only builds `hello-triangle`. So if you run the server from `target/generated` (`cd target/generated && python3 -m http.server`) the URL for hello-triangle should just be `localhost:8000` (or `localhost:8000/index.html`)

File: README.md (hunk 1 above)
Comment (kvark) [reply README.md:58]: agreed, can follow up :)

**Comment by rukai** [comment]
I entered only localhost:8000 into my browser. It was redirecting me to localhost:8000/examples

**Comment by grovesNL** [comment]
@rukai are you using the `index.html` from this PR?

The `index.html` in our `gh-pages` branch (https://wgpu.rs) will perform the automatic redirect to `examples` which isn't needed here.

File: README.md (hunk 1 above)
Comment (FuriouZz) [reply README.md:58]: Well, I do not really like the Chrome's translation popin when I develop a website. Bad habit ^^'.

**Comment by rukai** [comment]
Ahah that was exactly it, before this PR I was trying to get things working with the index.html from the gh-pages branch.
I accidentally left that old index.html lying around, and in my script to compile to wasm + copy the html file + run the webserver I was accidentally copying the wrong html file.

Thanks groves!

**Pull request merged**